app/controllers/admin/projects_controller.rb

```diff
@@ -29,6 +29,20 @@ def index
     end
   end
 
+  def update
+    @project = Project.find params[:id]
```

Comment:
Some places you use parentheses when calling methods some you don't, would be better to set a standard.